src/lib/Pagination.vue

```diff
@@ -202,7 +202,7 @@ export default {
   },
   watch: {
     current: function (page) {
-      this.$emit('change', page)
+      this.$dispatch('change', page)
```

Comment:
`$dispatch` is deprecated in [Vue 2](https://github.com/vuejs/vue/issues/2873). Besides, you can listen to component $emit events from the parent with `v-on`.